Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxRegEx/TextBoxRegex.Data.cs

```diff
@@ -34,7 +34,17 @@ public enum ValidationMode
             /// <summary>
             /// Update IsValid property with validation result and in case the textbox is not valid clear its value
             /// </summary>
-            Forced
+            Forced,
+
+            /// <summary>
+            /// Update IsValid property instantly with validation result at input and in case the textbox is not valid clear its value
+            /// </summary>
+            Instantly,
+
+            /// <summary>
+            /// Update IsValid property instantly with validation result and clear the newest character at input which is not valid
+            /// </summary>
+            InstantlyNumber
```

Comment:
Not sure if this is the right name for this. How about something like Dynamic or Latest

Reply:
Okay, I changed the name to Dynamic.